test/base/EscrowInitializeTest.js

```diff
@@ -1,45 +0,0 @@
-// JS Libraries
```

Comment:
Why did we remove this test? We still have the Escrow.initialize function

Reply:
It was actually removed since I added the `EscrowProxy` contract.